### PR TITLE
bump deps

### PR DESCRIPTION
**Pull request by pcunning**

### CODE REVIEW

File: scripts/greetings.coffee [hunk 1]
```diff
@@ -2,8 +2,14 @@ module.exports = (robot) ->
   enterReplies = require('../stubs/enterReplies')
   leaveReplies = require('../stubs/leaveReplies')  
   robot.enter (res) ->
-    res.send res.random enterReplies
+    console.log(robot.adapter.client.rtm.dataStore.getChannelGroupOrDMById res.message.room)
```
Comment (connerk) [inline scripts/greetings.coffee]: is this related to the bump, or pork barrel?

File: scripts/greetings.coffee [hunk 2]
```diff
@@ -2,8 +2,14 @@ module.exports = (robot) ->
...
+    if "testing_the_testbot" == robot.adapter.client.rtm.dataStore.getChannelGroupOrDMById res.message.room
+      robot.messageRoom res.message.user.id,  "This is your private welcome message"
+    else
+      res.send res.random enterReplies
+      
     
   robot.leave (res) ->
     res.send res.random leaveReplies
+    console.log(res.message.user.id)
```
Comment (connerk) [inline scripts/greetings.coffee]: console.log in production?

File: scripts/greetings.coffee (hunk 1 above)
Comment (pcunning) [reply scripts/greetings.coffee]: unrelated

Comment (connerk) [reply scripts/greetings.coffee]: I'm cool with it in one PR, but can it have a separate commit?
